.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Bug report
+about: Report a HTTPie bug
+title: ''
+labels: "new, bug"
+assignees: ''
+
+---
+
+**Checklist**
+
+- [ ] Bug can be reproduced
```

Comment:
I think 'Bug' over and over again is too droning&mdash;here's my suggestion:
```suggestion
- [ ] Reproducible
```

Reply:
@bl-ue `Issue is reproducible`?